cmd/root.go

```diff
@@ -60,6 +60,10 @@ func init() {
 	rootCmd.PersistentFlags().Uint("mfaRetrys", 3, "How often to retry TOTP Auth, only used in nointeractive modes")
 	rootCmd.PersistentFlags().Duration("mfaDelay", time.Second*10, "Delay between MFA Attempts, only used in noninteractive modes")
 
+	rootCmd.PersistentFlags().Bool("tlsSkipVerify", false, "Allow servers with self-signed certificates")
+	rootCmd.PersistentFlags().String("tlsClientPrivateKey", "", "Client private key for mtls")
+	rootCmd.PersistentFlags().String("tlsClientCert", "", "Client certificate for mtls")
```

Comment:
We probably should allow importing the Certificate and Key from a File Path like we do for userPrivateKeyFile.

Reply:
This is actually the case. I am going ot change the flag names and include the 'File' suffix.

Reply:
Oh, so its just Using the Path. This is a problem since other "File" Flags Import the File and save its contents in go-passbolt-cli's config instead of just paths to the Files when using the configure sub command. Id Prefer it if we don't have different behaviors between the File Flags.
Can you implement it like done here for the Key File? https://github.com/passbolt/go-passbolt-cli/blob/d9703ff6fd15d564d5112d95ac099b561d057af4/cmd/root.go#L107C1-L121C3

Reply:
I created a fileToContent function with the code you mentioned and used it for the client cert flags but also migrated the userPrivateKey reading step.